core/test/test-helpers.js

```diff
@@ -27,49 +29,67 @@ function timePasses(ms) {
   });
 }
 
-const flushCb = window.flush;
-window.flush = () =>
-  new Promise((resolve) => {
-    flushCb(() => {
-      if (window.requestIdleCallback) {
-        window.requestIdleCallback(() => resolve());
-      } else {
-        window.setTimeout(() => resolve(), 16);
-      }
-    });
-  });
+const flush = async () => {
+  utils.flush();
+  await timePasses(0);
+};
 
 function waitForEvent(el, event, times) {
   times = times || 1;
```

Comment:
Should we have a default value for the method param?

Reply:
Done, thanks!